udata_front/theme/gouvfr/assets/js/pages/PublishingForm/Step2DescribeDataset.vue

```diff
@@ -0,0 +1,456 @@
+                @change="(value) => dataset.license = value"
+                :allOption="$t('Select a license')"
+                :addAllOption="false"
+              />
+            </LinkedToAccordion>
+          </fieldset>
+          <fieldset class="fr-fieldset" aria-labelledby="time-legend">
+            <legend class="fr-fieldset__legend" id="time-legend">
+              <h2 class="subtitle subtitle--uppercase fr-mb-3v">
+                {{ $t("Time") }}
+              </h2>
+            </legend>
+            <div class="fr-fieldset__element">
+              <div class="fr-grid-row fr-grid-row--gutters">
+                <div class="fr-col-12 fr-col-md-6">
+                  <LinkedToAccordion
+                    :accordion="chooseFrequencyAccordionId"
+                    @blur="vWarning$.frequency.$touch"
+                  >
+                    <MultiSelect
+                      :placeholder="$t('Update frequency')"
+                      :searchPlaceholder="$t('Search a frequency...')"
+                      listUrl="/datasets/frequencies/"
+                      :values="dataset.frequency"
+                      @change="(value) => dataset.frequency = value"
+                      :required="true"
+                      :hasError="stateHasError('frequency')"
+                      :hasWarning="stateHasWarning('frequency')"
+                      :errorText="getErrorText('frequency')"
+                      :allOption="$t('Select an option')"
+                      :addAllOption="false"
+                    />
+                  </LinkedToAccordion>
+                </div>
+                <div class="fr-col-12 fr-col-md-6">
+                  <InputGroup
+                    :label="$t('Last update')"
+                    type="date"
+                    v-model="dataset.last_update"
+                  />
+                </div>
+              </div>
+            </div>
+            <LinkedToAccordion
+              :accordion="addTemporalCoverageAccordionId"
+              @blur="vWarning$.temporal_coverage.$touch"
+              class="fr-fieldset__element"
+            >
+              <InputGroup
+                :label="$t('Temporal coverage')"
+                type="range"
+                v-model="dataset.temporal_coverage"
+              />
+            </LinkedToAccordion>
+          </fieldset>
+          <fieldset class="fr-fieldset" aria-labelledby="space-legend">
+            <legend class="fr-fieldset__legend" id="space-legend">
+              <h2 class="subtitle subtitle--uppercase fr-mb-3v">
+                {{ $t("Space") }}
+              </h2>
+            </legend>
+            <LinkedToAccordion
+              :accordion="addSpatialInformationAccordionId"
+              @blur="vWarning$.spatial.$touch"
+              class="fr-fieldset__element"
+            >
+              <div class="fr-grid-row fr-grid-row--gutters">
+                <div class="fr-col-12 fr-col-md-6">
+                  <MultiSelect
```

Comment:
It would be nice to show additional properties (insee code, geography level, etc.) in the suggest list, as is done in current admin. It may be done after GeoZone refactoring.

Reply:
I'm adding this on Figma as comment

Reply:
Tracked by https://github.com/etalab/data.gouv.fr/issues/1196